l10n/en/navigation_refresh.ftl

```diff
@@ -19,8 +19,10 @@ navigation-refresh-innovation-projects-v2 = Innovation Projects
 navigation-refresh-blog = Blog
 navigation-refresh-our-mission = Our Mission
 navigation-refresh-our-work = Our Work
+
+# Obsolete string (expires: 2025-01-05)
 navigation-refresh-mozilla-ai = { -brand-name-mozilla } AI
-navigation-refresh-mozilla-ai-v2 = { -brand-name-mozilla-ai }
+navigation-refresh-mozilla-ai-v2 = { -brand-name-mozilla-ai-v2 }
 navigation-refresh-mozilla-ventures = { -brand-name-mozilla } Ventures
```

Comment:
Hmm, this may actually need:

```suggestion

# Obsolete string (expires: 2025-01-05)
navigation-refresh-mozilla-ai-v2 = { -brand-name-mozilla-ai }

navigation-refresh-mozilla-ai-v3 = { -brand-name-mozilla-ai-v2 }

# Obsolete string (expires: 2025-01-05)
navigation-refresh-mozilla-ventures = { -brand-name-mozilla } Ventures
```

so that would mean `navigation-refresh-mozilla-ai-v3` to the submenu html?

Reply:
hmm true... what a mess 😄